src/NuGet.Clients/PackageManagement.VisualStudio/Migrate/ProjectJsonToPackageRefMigrator.cs

```diff
@@ -22,23 +24,34 @@ public static class ProjectJsonToPackageRefMigrator
             var projectJsonFilePath = ProjectJsonPathUtilities.GetProjectConfigPath(Path.GetDirectoryName(project.MSBuildProjectPath),
                 Path.GetFileNameWithoutExtension(project.MSBuildProjectPath));
 
+            if (!File.Exists(projectJsonFilePath))
+            {
+                throw new FileNotFoundException(string.Format(Strings.Error_FileNotExists, projectJsonFilePath));
```

Comment:
I'm curious what your code does if you remove this line of code. Presumably you'd throw the same exception, just later on. Is that right? I hope you don't crash.

I bring this up because VS crashes sometimes when it's doing a package restore (automatically) and I'm switching branches in git from the command line. It makes me wonder how well nuget handles unexpected situations like that.